nashville-software-school/bangazon-llc
# Updated Chinook setup and instructions in Ch1 of Book 3

### Changes Made

* Added context re: _what even is chinook?_ 
* Added GIF demonstrating how to create a repo in cohort's GH org
* Clarified setup and cleaned up instructions/exercise prompts
* Added information to help students know whether their query is sufficient or not

### Steps To Test

1. [View this branch on GitHub](https://github.com/nashville-software-school/bangazon-llc/tree/mp-chinook-setup)
1. Review Chinook intro - it should be concise/informative without being overwhelming.
1. Review GH repo GIF - it should show how to create a repo on the Cohort's GitHub
1. Read setup instructions and exercise prompts - should provide context for whether student's queries are sufficient or not
